tools/build_defs.bzl

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2017 streaml.io .
```

Comment:
Again this should be in tools/rules as well - until we move it later versions of bazel.